kubernetes/linux/setup.sh

```diff
@@ -67,11 +67,11 @@ chmod 544 /opt/telegraf
 # Use wildcard version so that it doesnt require to touch this file
 /$TMPDIR/docker-cimprov-*.*.*-*.*.sh --install
 
-#download and install fluent-bit(td-agent-bit)
+#download and install fluent-bit(fluent-bit)
 wget -qO - https://packages.fluentbit.io/fluentbit.key | sudo apt-key add -
 sudo echo "deb https://packages.fluentbit.io/ubuntu/bionic bionic main" >> /etc/apt/sources.list
 sudo apt-get update
-sudo apt-get install td-agent-bit=1.7.8 -y
+sudo apt-get install fluent-bit=2.0.5 -y
```

Comment:
are there any changes to the behavior or functionality in 2.0.5 compared to 1.7.8 for both Linux & windows?

Reply:
<img width="1418" alt="cpu" src="https://user-images.githubusercontent.com/10297394/209891038-494bea49-4859-4ad3-8912-e165c3c1aa45.png">
<img width="1429" alt="memory" src="https://user-images.githubusercontent.com/10297394/209891041-fa9921be-01ee-4596-9a49-41e417cd2447.png">

Reply:
same comparison for v1 route

cpu
<img width="1421" alt="cpuv1" src="https://user-images.githubusercontent.com/10297394/209900941-acd4f131-9903-4107-95b5-3d68c219723b.png">

memory
<img width="1432" alt="memoryv1" src="https://user-images.githubusercontent.com/10297394/209900939-f8567425-597a-4572-aefe-2f8a93488849.png">

Reply:
Thanks for sharing these. do we have the CPU and memory charts with multi-line for container log v2?

Reply:
I already had shared those results. will share again on teams